week-2-practise/level-2/5-indexOf.js

```diff
@@ -5,7 +5,8 @@
 // TIP: Use the .indexOf() method
 
 function remove(arr, valueToRemove) {
-  return; // complete this statement
+  arr.splice(arr.indexOf(valueToRemove), 1);
+  return arr; // complete this statement
```

Comment:
Same as above. You have to return a *new* array and not modify the original one.